huaweicloud/utils/validators.go

```diff
@@ -226,22 +156,6 @@ func ValidateVBSBackupName(v interface{}, k string) (ws []string, errors []error
 	return
 }
 
-//lintignore:V001
-func ValidateVBSBackupDescription(v interface{}, k string) (ws []string, errors []error) {
-	value := v.(string)
-	if len(value) > 64 {
-		errors = append(errors, fmt.Errorf(
-			"%q cannot be longer than 64 characters: %q", k, value))
-	}
-	pattern := `^[^<>]+$`
-	if !regexp.MustCompile(pattern).MatchString(value) {
-		errors = append(errors, fmt.Errorf(
-			"%q doesn't comply with restrictions (%q): %q",
-			k, pattern, value))
-	}
-	return
-}
-
 //lintignore:V001
```

Comment:
Why not move the remaining validations to the common directory?

Reply:
Other validations are not public or can be replaced by existing hashicorp public functions, and will be modified by another PR in the future.